docs/reference/cat/recovery.asciidoc

```diff
@@ -13,6 +13,8 @@ to the <<indices-recovery, indices recovery>> API.
 
 `GET /_cat/recovery/<index>`
 
+`GET /_cat/recovery
```

Comment:
Missing ` at the end

Reply:
Thanks for catching this @tlrx. Fixed with cc118f7.